builder/manifest.json

```diff
@@ -602,6 +609,76 @@
         ]
       }
     },
+    {
+      "id": "dashboardv2-base",
+      "base": "go",
+      "layers": [{
+        "script": "builder/img/dashboardv2-base.sh",
+        "inputs": [
+          "controller/api/*.proto"
+        ],
+        "limits": { "temp_disk": "1G" }
+      },
+      {
+        "gobin": ["github.com/go-bindata/go-bindata/go-bindata"]
+      }]
+    },
+    {
+      "id": "dashboardv2-compiled",
+      "base": "dashboardv2-base",
+      "layers": [
+        {
+         "protobuildjs": {
+           "controller/api": "dashboardv2/src/generated"
+          }
+        },
+        {
+         "script": "dashboardv2/img/compile.sh",
+         "inputs": [
+           "dashboardv2/.eslintignore",
+           "dashboardv2/yarn.lock",
+           "dashboardv2/*.json",
+           "dashboardv2/*.ts",
+           "dashboardv2/config/*.js",
+           "dashboardv2/scripts/*.js",
+           "dashboardv2/public/*.png",
+           "dashboardv2/public/*.ico",
+           "dashboardv2/public/*.json",
+           "dashboardv2/public/*.html",
+           "dashboardv2/src/*.css",
+           "dashboardv2/src/*.ts",
+           "dashboardv2/src/*.tsx",
+           "dashboardv2/src/util/*.ts",
+           "dashboardv2/src/generated/*.ts",
+           "dashboardv2/src/generated/*.js",
+           "dashboardv2/src/generated/google/api/*.ts",
+           "dashboardv2/src/generated/google/api/*.js",
+           "dashboardv2/src/KeyValueEditor/*.ts",
+           "dashboardv2/src/KeyValueEditor/*.tsx"
+         ],
+         "limits": { "memory": "3G", "temp_disk": "3G" }
+       }
+      ]
+    },
+    {
+      "id": "dashboardv2",
```

Comment:
Assuming we're no longer going to work on the old dashboard, should we just call this one `dashboard` now and remove the old build (we can always reference the old dashboard layers from the TUF repo if we want to use them)